src/storage/mod.rs

```diff
@@ -205,6 +307,46 @@ where
         }
     }
 
+    /// Returns an entry to the component associated to the entity.
+    ///
+    /// Behaves somewhat similarly to `std::collections::HashMap`'s entry api.
+    ///
+    ///## Example
```

Comment:
missing space

Reply:
That's strange; our other examples have spaces, too.

Reply:
Alright, seems to work now?